<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Directions">
      <property name="text">
       <string>Enter the coefficients for a quadratic of the form: Ax^2 + Bx + C = 0</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QLabel" name="C_label">
        <property name="text">
         <string>C:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="A_label">
        <property name="text">
         <string>A:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="A_text"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="B_text"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="C_text"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="B_label">
        <property name="text">
         <string>B:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculate_button">
      <property name="text">
       <string>Calculate Roots</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="x1_label">
        <property name="text">
         <string>Solutions: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="root2_label">
        <property name="text">
         <string>      </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="root1_label">
        <property name="text">
         <string>   </string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="root1_im_label">
        <property name="text">
         <string>     </string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="root2_im_label">
        <property name="text">
         <string>     </string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calculate_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CalculateButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CalculateButtonClicked()</slot>
 </slots>
</ui>
